external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -376,8 +376,7 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationSuite with SharedSQLCo
     val e = intercept[org.apache.spark.SparkException] {
       spark.read.jdbc(jdbcUrl, "tableWithCustomSchema", new Properties()).collect()
     }
-    assert(e.getMessage.contains(
-      "requirement failed: Decimal precision 39 exceeds max precision 38"))
+    assert(e.getMessage.contains("Decimal precision 39 exceeds max precision 38"))
```

Comment:
as a very nit, I'd rather check the exception type, which I think is more important than the exact message. Now we should be coherent in the whole codebase and always throw an `ArithmeticException`, while previously we were sometimes throwing `RuntimeException` or others for the same case.

Reply:
Thank you for review, @mgaido91 .

Sure, of course, we can check the underlying exception type by `e.getCause` from `SparkException` additionally. I'll add that.

However, `message checking` is a more fine-grained verification. As you know, `ArithmeticException` and `ParseException` are not specific. For example, `ArithmeticException` can be caused by `divide by zero`. We should check the error message always.